File: 3ºCurso/2º Cuatrimestre/SD/Practica3/ServivioSudoku/ServivioSudoku/DatosSudoku.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ServicioSudoku
{
    class DatosSudoku
    {
        public int codigo;
        public char[,] Tablero = new char[9,9];
        public TipoDificultad Dificultad;
        public int llamada = 0;
        public DatosSudoku(int cod)
        {
            this.codigo = cod;
        }
    }
}

File: 3ºCurso/2º Cuatrimestre/SD/Practica2/SudokuRMIComun/src/Estructuras/DatosSudoku.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Estructuras;

import Estructuras.TipoDificultad;

/**
 *
 * @author carlo
 */
public class DatosSudoku {
    
    public int codigo;
    public char[][] Tablero=new char[9][9];
    public TipoDificultad Dificultad;
    public int llamada=0;
    public DatosSudoku(int cod) {
        this.codigo=cod;
    }
    
}
